graphics/mupdf/Portfile

```diff
@@ -20,49 +19,71 @@ long_description    MuPDF is a lightweight PDF and XPS viewer and \
                     parser/rendering library. \
                     \
                     The renderer in MuPDF is tailored for high-quality \
-                    anti-aliased graphics. It renders text with metrics and \
-                    spacing accurate to within fractions of a pixel for the \
-                    highest fidelity in reproducing the look of a printed \
-                    page on screen. \
+                    anti-aliased graphics. It renders text with metrics \
+                    and spacing accurate to within fractions of a pixel \
+                    for the highest fidelity in reproducing the look \
+                    of a printed page on screen. \
                     \
                     MuPDF is also small, fast, and yet complete. It supports \
                     PDF 1.7 with transparency, encryption, hyperlinks, \
                     annotations, search and many other bells and whistles. \
                     MuPDF can also read XPS documents (OpenXPS ECMA-388).
 
-homepage            https://www.mupdf.com/
+homepage            https://www.mupdf.com
 
-master_sites        ${homepage}downloads/archive/
+master_sites        ${homepage}/downloads/archive/
 distname            ${name}-${version}-source
 
-checksums           rmd160  63d39c7cfc70ac08beed28d8da6e581bbf09ae6f \
-                    sha256  deb0347a3f9ce0967611d479a8a07b57e2c730b0aaf09dce28ee41abb9ac4838 \
-                    size    55146083
+checksums           rmd160  e19bad3ca13762fe5b238093d62e5ce292b69df4 \
+                    sha256  7fbced6d6024608661d773b93990119db7eb2483f0d3e28855eadef9ca9a8686 \
+                    size    55313640
 
-depends_build       port:pkgconfig
+depends_build-append \
+                    bin:pkg-config:pkgconfig
```

Comment:
as @ryandesign pointed out in a different PR; it seems like one should use "path/bin:....". I presume that that syntax will only check for things in "${prefix}/bin"

Reply:
Yes, this should be changed, will do it today.